corehq/form_processor/backends/couch/processor.py

```diff
@@ -102,8 +102,8 @@ def assign_new_id(cls, xform):
         return xform
 
     @classmethod
-    def xformerror_from_xform_instance(self, instance, error_message, with_new_id=False):
-        return XFormError.from_xform_instance(instance, error_message, with_new_id=with_new_id)
+    def xformerror_from_xform_instance(self, instance, error_message, replace_form_id=True):
+        return XFormError.from_xform_instance(instance, error_message, replace_form_id=replace_form_id)
```

Comment:
could be slightly confusing that this defaults to `True` but the arg in `from_xform_instance` defaults to `False`. I'd recommend making them both default to the same thing.